nuxeo-lambda-image-conversion/src/test/java/org/nuxeo/lambda/image/conversion/common/MemLambdaCaller.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * (C) Copyright 2006-2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ * Contributors:
+ *     anechaev
+ */
+package org.nuxeo.lambda.image.conversion.common;
+
+import org.codehaus.jettison.json.JSONArray;
+import org.codehaus.jettison.json.JSONObject;
```

Comment:
Hmm I see it was already used in other files of nuxeo-lambda previously, but please don't use `org.codehaus.jettison`. Our JSON library should be standardized on Jackson 2 only (could be refactored in a separate commit in all of nuxeo-lambda).

Reply:
https://jira.nuxeo.com/browse/NXP-25023